azurerm/internal/services/netapp/resource_arm_netapp_snapshot.go

```diff
@@ -94,9 +97,11 @@ func resourceArmNetAppSnapshotCreate(d *schema.ResourceData, meta interface{}) e
 	}
 
 	location := azure.NormalizeLocation(d.Get("location").(string))
+	t := d.Get("tags").(map[string]interface{})
 
 	parameters := netapp.Snapshot{
 		Location: utils.String(location),
+		Tags:     tags.Expand(t),
```

Comment:
_minor_ no need to change it but this is preferred
```suggestion
		Tags: tags.Expand(d.Get("tags").(map[string]interface{})),
```